packages/models-library/src/models_library/services_metadata_editable.py

```diff
@@ -33,7 +33,9 @@ class ServiceMetaDataEditable(ServiceBaseDisplay):
         "If now>=deprecated, the service is retired",
     )
     classifiers: list[str] | None
-    quality: dict[str, Any] = {}
+    quality: dict[str, Any] = Field(
+        default_factory=dict, json_schema_extra={"default": {}}
```

Comment:
@pcrespov question, is this necessary? is this used in a json schema? well I guess it does hurt